rasa/core/channels/channel.py

```diff
@@ -9,6 +9,7 @@
 from typing import Text, List, Dict, Any, Optional, Callable, Iterable, Awaitable
 
 import rasa.utils.endpoints
+from rasa.cli.utils import button_to_string
```

Comment:
thanks for moving this funciton to the right spot. can you import the module instead of the function? e.g.
```
from rasa.cli import utils as cli_utils
... 
cli_utils.button_to_string()
```